plugins/kafka/kafka.hpp

```diff
@@ -272,11 +272,13 @@ extern "C++"
                  */
                 void commitOffset(__int64 offset) const;
 
+#ifndef _CONTAINERIZED
                 /**
                  * If the offset file does not exist, create one with a
                  * default offset
                  */
-                void initFileOffsetIfNotExist() const;
+                void _initFileOffsetIfNotExist() const;
```

Comment:
looks like this is what's causing the compile error issues (in BM).

Reply:
Well that was a rookie mistake. Fixed.